### PR TITLE
images: add Fedora-39 image

### PR DESCRIPTION
**Pull request by tomasmatus**
* [x] image-refresh fedora-39

### CODE REVIEW

**Comment by martinpitt** [comment]
Wohoo, thanks! I fixed the version in the task, and set it off. I was going to complain about "plz copy naughties", but some good soul already did that (presumably for packit tests). So, :crossed_fingers: !

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/fedora-39-20230823-141717.log

File: images/scripts/fedora-39.setup [hunk 1]
```diff
@@ -0,0 +1 @@
+fedora.setup
```
Comment (martinpitt) [inline images/scripts/fedora-39.setup:1]: This is a plain file; it must be a *symlink* to fedora.setup.

**Comment by cockpituous** [comment]
image-refresh fedora-39 done: https://github.com/cockpit-project/bots/commits/image-refresh-fedora-39-20230823-145013

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/fedora-39-20230823-145050.log

**Comment by martinpitt** [comment]
Triggering starter-kit. That was missing in testmap in the recent PR. But if/once it succeeds (very likely), let's just add it to main straight away.

**Review by martinpitt — APPROVED** [review]
Cheers!

**Pull request merged**